requirements.txt

```diff
@@ -20,10 +21,17 @@ mock==2.0.0
 num2words==0.5.6
 ofxparse==0.16
 passlib==1.6.5
+<<<<<<< HEAD
 Pillow==5.4.0
 polib==1.1.0
+=======
+Pillow==4.0.0 ; python_version < '3.7'
+Pillow==6.1.0 ; python_version >= '3.7'
+>>>>>>> f645d752572... temp
```

Comment:
I suggest to edit as follows:
```
Pillow==5.4.0 ; python_version < '3.7'
Pillow==6.1.0 ; python_version >= '3.7'
polib==1.1.0
```

Reply:
Finally, I changed
Pillow==6.1.0 ; python_version >= '3.7'
to
Pillow==6.1.0 ; sys_platform == 'win32' and python_version >= '3.7'

because Odoo 13.0 is built against Debian Buster for which [default Python version is 3.7](https://packages.debian.org/buster/python3) and [python3-pil](https://packages.debian.org/buster/python3-pil) package is 5.4.1

We try to stick to Debian package versions.